app/controllers/physical_server_controller.rb

```diff
@@ -33,7 +33,7 @@ def show_list
   end
 
   def textual_group_list
-    []
+    [%i(properties relationships assets)]
```

Comment:
looks like we also have https://github.com/ManageIQ/manageiq-ui-classic/pull/1294 that can address the above.